py-polars/polars/series/datetime.py

```diff
@@ -944,6 +944,82 @@ def nanosecond(self) -> Series:
 
         """
 
+    def minute_of_day(self) -> Series:
+        """
+        Extract the minute of the day from the underlying DateTime representation.
+
+        Applies to Datetime columns.
+
+        Returns the minute number from 0 to 1439.
+
+        Returns
+        -------
+        Series
+            Series of data type :class:`UInt32`.
+
+        Examples
+        --------
+        >>> from datetime import datetime
+        >>> start = datetime(2001, 1, 1)
+        >>> stop = datetime(2001, 1, 1, 1, 4, 0)
+        >>> date = pl.date_range(start, stop, interval="2m", eager=True)
+        >>> date
+        shape: (3,)
+        Series: 'date' [datetime[μs]]
+        [
+                2001-01-01 00:00:00
+                2001-01-01 00:02:00
+                2001-01-01 00:04:00
+        ]
+        >>> date.dt.minute_of_day()
+        shape: (3,)
+        Series: 'date' [u32]
+        [
+                0
+                2
+                64
```

Comment:
hi - could you clarify how `2001-01-01 00:04:00` corresponds to `minute of the day` being `64`? Not sure I follow

Reply:
Apologies this was a typo - the example was meant to be `2001-01-01 01:04:00`